src/ScrollContainer/index.tsx

```diff
@@ -1,6 +1,8 @@
 import * as React from 'react';
 import { Scrollbar, ScrollbarProps } from 'react-scrollbars-custom';
 
+import omit = require('lodash/omit');
```

Comment:
```suggestion
import { omit } = from 'lodash';
```
We ditched the require-equals syntax.

Reply:
I saw that in other components of ui-kit and therefore used the same but agree with that.

Reply:
Yeah, it's just not updated yet.